Brokerages/Zerodha/ZerodhaBrokerage.cs

```diff
@@ -982,8 +983,6 @@ private IEnumerable<BaseData> GetHistoryForPeriod(Symbol symbol, DateTime start,
                     $"from {start:s} to {end:s}"));
             }
 
-            var period = resolution.ToTimeSpan();
-
             foreach (var candle in candles)
             {
                 yield return new TradeBar(candle.TimeStamp.ConvertFromUtc(TimeZones.Kolkata),symbol,candle.Open,candle.High,candle.Low,candle.Close,candle.Volume,resolution.ToTimeSpan());
```

Comment:
`.ConvertFromUtc(TimeZones.Kolkata)` TZ here should also be driven by the exchange TZ, the history request has this information